owncloudApp/src/main/java/com/owncloud/android/presentation/ui/files/filelist/MainFileListFragment.kt

```diff
@@ -41,20 +49,32 @@ import com.owncloud.android.presentation.ui.files.SortOptionsView
 import com.owncloud.android.presentation.ui.files.SortOrder
 import com.owncloud.android.presentation.ui.files.SortType
 import com.owncloud.android.presentation.ui.files.ViewType
+import com.owncloud.android.presentation.ui.files.createfolder.CreateFolderDialogFragment
+import com.owncloud.android.presentation.viewmodels.files.FilesViewModel
 import com.owncloud.android.ui.activity.FileListOption
 import com.owncloud.android.utils.FileStorageUtils
 import org.koin.androidx.viewmodel.ext.android.viewModel
+import org.koin.java.KoinJavaComponent.get
 
-class MainFileListFragment : Fragment(), SortDialogListener, SortOptionsView.SortOptionsListener, SortOptionsView.CreateFolderListener {
+class MainFileListFragment : Fragment(), SortDialogListener, SortOptionsView.SortOptionsListener, SortOptionsView.CreateFolderListener,
+    CreateFolderDialogFragment.CreateFolderListener {
 
     private val mainFileListViewModel by viewModel<MainFileListViewModel>()
 
+    val ARG_LIST_FILE_OPTION = MainFileListFragment::class.java.canonicalName +
```

Comment:
I think we should add this argument in the companion object like the other arguments, also, you can use the same structure as others to concatenate the strings. what do you think?

Reply:
Sure!